src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -672,10 +672,10 @@ private ServerEncryptionOptions applyConfigInternal()
             if (require_client_auth && (internode_encryption == InternodeEncryption.rack || internode_encryption == InternodeEncryption.dc))
             {
                 logger.warn("Setting require_client_auth is incompatible with 'rack' and 'dc' internode_encryption values."
-                          + " It is possible for an internode connection to pretend to be in the same rack/dc by spoofing"
-                          + " its broadcast address in the handshake and bypass authentication. To ensure that mutual TLS"
-                          + " authentication is not bypassed, please set internode_encryption to 'all'. Continuing with"
-                          + " insecure configuration.");
+                            + " It is possible for an internode connection to pretend to be in the same rack/dc by spoofing"
```

Comment:
The same as above - as the formatting is not related to the change itself it is worth excluding it from the PR to avoid drawing attention to these lines.